src/backend/types.ml

```diff
@@ -105,7 +105,9 @@ module Kind = struct
     | Strictly
     | JJ_Strictly
     | Jack_and_Jill
-  [@@deriving yojson]
+  [@@deriving yojson, enum, show]
+
+  let all = List.filter_map of_enum (List.init (to_enum Jack_and_Jill + 1) Fun.id)
```

Comment:
I'm not sure this is simpler than the current code, particularly for such a simple and small type...

Reply:
ping on that @But2ene